webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -31,11 +30,17 @@ export type Props = {
     submitLabel?: string;
     notifyOnCancel?: boolean;
     state?: string;
-    onExited?: () => void;
+    emojiMap: EmojiMap;
+};
+
+export type MapDispatchToProps = {
     actions: {
         submitInteractiveDialog: (submission: DialogSubmission) => Promise<ActionResult<SubmitDialogResponse>>;
     };
-    emojiMap: EmojiMap;
+}
+
+export interface Props extends MapStateToProps, MapDispatchToProps {
```

Comment:
To fix the above, we could just say this is `Partial`:
```suggestion
export interface Props extends Partial<MapStateToProps>, MapDispatchToProps {
```

Reply:
changes done